From 1d7df8402d00bfb5958c707d22672efae48d8cbb Mon Sep 17 00:00:00 2001
From: Salvo Giangreco <giangrecosalvo9@gmail.com>
Date: Sun, 31 Aug 2025 15:42:41 +0200
Subject: [PATCH] Allow custom CONNECTION_PERSONALIZATION value

UN1CA: this patch is not complete! Please check unica/patches/product_feature/customize.sh for more info.

Change Id: 841857e6d0a2a9d587d04307a6f29a1688c3245d Mon Sep 17 00:00:00 2001
From: Erick-04 <solonsosua@gmail.com>
Date: Sun, 4 Oct 2026 17:11:58 +0200
Subject: [PATCH] Allow custom CONNECTION_PERSONALIZATION value

---
 smali/com/samsung/android/server/wifi/SemWifiInjector.smali | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/smali/com/samsung/android/server/wifi/SemWifiInjector.smali b/smali/com/samsung/android/server/wifi/SemWifiInjector.smali
index 6adc0f1..58d69fa 100644
--- a/smali/com/samsung/android/server/wifi/SemWifiInjector.smali
+++ b/smali/com/samsung/android/server/wifi/SemWifiInjector.smali
@@ -1866,6 +1866,8 @@
 
     iput-object v1, v4, Lcom/samsung/android/server/wifi/SemWifiInjector;->mWifiCoexManager:Lcom/samsung/android/server/wifi/SemWifiCoexManager;
 
+    const-string v0, "CONFIG_CONNECTION_PERSONALIZATION"
+
     invoke-static {v0}, Ljava/lang/Integer;->parseInt(Ljava/lang/String;)I
 
     move-result v0
-- 
2.51.0

